rusk-wallet/Cargo.toml

```diff
@@ -42,5 +42,7 @@ dusk-bls12_381-sign = { version = "0.3.0-rc", default-features = false, features
 
 rusk-abi = { path = "../rusk-abi" }
 
+rusqlite = "0.26"
```

Comment:
Why on it's own? Just curious if there's a reason. If not, I'd put it with the rest of the 3rd-party crates.